docs/deploy/index.md

```diff
@@ -143,6 +144,12 @@ kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/cont
 kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/controller-v0.35.0/deploy/static/provider/do/deploy.yaml
 ```
 
+#### Scaleway
+
+```console
+kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/controller-v0.35.0/deploy/static/provider/scw/deploy.yaml
```

Comment:
this will not work because the tag already exists. Please use master (I will update that in the next release)

Reply:
true :sweat_smile: fixed :)